WordPress/src/main/java/org/wordpress/android/ui/posts/EditPostActivity.java

```diff
@@ -404,6 +406,40 @@ public void run() {
         ActivityId.trackLastActivity(ActivityId.POST_EDITOR);
     }
 
+    // this method aims at recovering the current state of media items if they're inconsistent within the PostModel.
+    private void resetUploadingMediaToFailedIfNotInProgressOrQueued(Context context, PostModel post) {
+        boolean useAztec = AppPrefs.isAztecEditorEnabled();
+        if (useAztec && context != null && post != null) {
+
+            String oldContent = post.getContent();
+            if (!UploadService.hasPendingOrInProgressMediaUploadsForPost(post) &&
+                    ( AztecEditorFragment.hasMediaItemsMarkedUploading(context, oldContent)
+                            || AztecEditorFragment.hasMediaItemsMarkedFailed(context, oldContent))) {
```

Comment:
This is one of the cases described above: there could be a possibility that an initially failed item that was retried, has now a remote URL in the db but was still not unmarked in the Post body before the app stopped. Curious situation, but totally possible.
This case is specifically addressed and [commented in code here](https://github.com/wordpress-mobile/WordPress-Android/pull/6460/files#diff-ffbdd6cd0e03b7a13fcdb13666c6d8c3R1538)
Added another comment here just in case 2cb38f7